Dockerfile

```
@@ -1,102 +1,103 @@
 
 WORKDIR /app
 
-# Set a default version for builds - will be overridden by build args
-ENV APP_VERSION=0.1.0-docker
+# Set Mix environment and a default application version (overridable via build-arg)
+ENV MIX_ENV=prod \
+    APP_VERSION=0.1.0-docker
 
-# Install build tools
+# Install only the build tools we need
 RUN apt-get update \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)